docusaurus.config.js

```diff
@@ -112,6 +112,58 @@ const config = {
         contextualSearch: true,
       },
     }),
+
+    plugins: [
+      [
+        '@docusaurus/plugin-client-redirects',
+        {
+          redirects: [
+            // Mobile tracker v3 redirects
+            {
+              from: '/docs/collecting-data/collecting-from-own-applications/mobile-trackers/mobile-trackers-v3-0',
```

Comment:
Could you create a separate file to keep this and import it here somehow? I am concerned with the size of `docusaurus.config.js` getting out of hand once we start adding more redirects as we move things around

Reply:
Makes sense, extracted it now